pkg/kubelego/watch.go

```diff
@@ -70,15 +116,32 @@ func (kl *KubeLego) WatchEvents() {
 				return
 			}
 			kl.Log().Debugf("CREATE ingress/%s/%s", addIng.Namespace, addIng.Name)
-			kl.workQueue.Add(true)
```

Comment:
Ok I get it. So previously we added a bool to the queue!?  which meant that all certificates were re-validated every time?

Reply:
Yes spot on 😬